docker-compose.yml

```diff
@@ -86,10 +99,9 @@ services:
       RUN_SIDEKIQ_IN_TEST_MODE: true
       ENABLE_SYNC_TRIGGER_ENDPOINT: true
```

Comment:
do we need these env variables if not using trigger endpoints?

Reply:
Nope! Good spot, I have removed.